pkg/constants/constants.go

```diff
@@ -21,7 +21,7 @@ const (
 	MaxTrafficPercent = 100
 	MaxTrafficValue   = 10000
 	StatusRunning     = "RUNNING"
-	SDKVersion        = "1.0.0"
+	SDKVersion        = "1.1.0"
```

Comment:
let's make it `1.8.0` as other SDKs promote this from this version onwards. Similarly in CHANGELOG.md file

Reply:
@softvar will need testing too as like dotnet it is strict typed so missing sdk key can cause issues

Reply:
ok sir?